packages/exchange-ui-core/src/containers/Exchange.tsx

```diff
@@ -150,7 +152,8 @@ export function Exchange(props: IProps) {
                     }
 
                     const newGenerationDateStart = values.generationDateStart
-                        ?.startOf('month')
+                        ?.utcOffset(Number(environment.MARKET_UTC_OFFSET), true)
+                        .startOf('month')
```

Comment:
Do we need to offset here? I think this date is already in MARKET_UTC_OFFSET from Formik datepicker.

Reply:
it is separate changes for exchange ui and origin ui, it have to be here

Reply:
it is separate changes for exchange ui and origin ui, it have to be here